lib/Views/src/layouts/DateTimeView64x64.h

```diff
@@ -94,218 +92,61 @@ class DateTimeView64x64 : public IDateTimeView
-    {
-        m_dayOffColor = color;
-        updateLampWidgetsColors();
-    }
+    DateTimeView64x64(const DateTimeView64x64& other);
+    DateTimeView64x64& operator=(const DateTimeView64x64& other);
 
     /**
-     * Set weekday indicator depended on the given time info.
+     * Draw analog clock backround (the minute tick marks)
      *
-     * @param[in] timeInfo the current time info.
-     */
-    void setWeekdayIndicator(tm timeInfo) override;
-
-    /** Max. number of lamps. One lamp per day in a week. */
-    static const uint8_t    MAX_LAMPS       = 7U;
-
-protected:
-
-    /** Distance between two lamps in pixel. */
-    static const uint8_t    LAMP_DISTANCE   = 1U;
-
-    /** Lamp width in pixel. */
-    static const uint8_t    LAMP_WIDTH      = (CONFIG_LED_MATRIX_WIDTH - ((MAX_LAMPS + 1U) * LAMP_DISTANCE)) / MAX_LAMPS;
-
-    /** Lamp distance to the canvas border in pixel. */
-    static const uint8_t    LAMP_BORDER     = (CONFIG_LED_MATRIX_WIDTH - (MAX_LAMPS * LAMP_WIDTH) - ((MAX_LAMPS - 1U) * LAMP_DISTANCE)) / 2U;
-
-    /** Lamp height in pixel. */
-    static const uint8_t    LAMP_HEIGHT     = 1U;
-
-    /** Lamp 0 x-coordinate in pixel. */
-    static const uint8_t    LAMP_0_X        = LAMP_BORDER + (0 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 1 x-coordinate in pixel. */
-    static const uint8_t    LAMP_1_X        = LAMP_BORDER + (1 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 2 x-coordinate in pixel. */
-    static const uint8_t    LAMP_2_X        = LAMP_BORDER + (2 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 3 x-coordinate in pixel. */
-    static const uint8_t    LAMP_3_X        = LAMP_BORDER + (3 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 4 x-coordinate in pixel. */
-    static const uint8_t    LAMP_4_X        = LAMP_BORDER + (4 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 5 x-coordinate in pixel. */
-    static const uint8_t    LAMP_5_X        = LAMP_BORDER + (5 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp 6 x-coordinate in pixel. */
-    static const uint8_t    LAMP_6_X        = LAMP_BORDER + (6 * (LAMP_WIDTH + LAMP_DISTANCE));
-
-    /** Lamp y-coordindate in pixel. */
-    static const uint8_t    LAMP_Y          = CONFIG_LED_MATRIX_HEIGHT - 1U;
-
-    /**
-     * Text width in pixels.
-     */
-    static const uint16_t   TEXT_WIDTH      = CONFIG_LED_MATRIX_WIDTH;
-
-    /**
-     * Text height in pixels.
-     */
-    static const uint16_t   TEXT_HEIGHT     = CONFIG_LED_MATRIX_HEIGHT - LAMP_HEIGHT;
-
-    /**
-     * Text widget x-coordinate in pixels.
+     * @param[in] gfx   Graphic functionality to draw on the underlying canvas.
      */
-    static const int16_t    TEXT_X          = 0;
+    void drawAnalogClockBackground(YAGfx& gfx);
 
     /**
-     * Text widget y-coordinate in pixels.
+     * Draw analog clock hands for given time.
+     *
+     * @param[in] gfx   Graphic functionality to draw on the underlying canvas.
+     * @param[in] now current time

```

Comment:
Move "current Time" unter "Graphic ..." and start it with uppercase.

Reply:
Was actually a redundant 2nd comment block for the method (now removed).
Done with next commit.